jkva-rakudo-perl6/run.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
```

Comment:
You could also merge this into the docker `CMD`

Reply:
Did that, and removed the p6 build script. Thanks!